### PR TITLE
Move `local_global_conversion`

### PR DESCRIPTION
**Pull request by Evang264**
Moved
* `local_global_conversion.py`
* `drone_odometry_local.py`

To the common repository. Updated references to the two files.

### CODE REVIEW

**Comment by Evang264** [comment]
Still waiting for my PR in the common repository to be approved first before the unit tests can pass

File: modules/flight_interface/flight_interface.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 Creates flight controller and combines odometry data and timestamp.
 """
 
-from . import local_global_conversion
-from .. import drone_odometry_local
+from ..common.mavlink.modules import local_global_conversion
+from ..common.mavlink.modules import drone_odometry_local
```
Comment (Xierumeng) [inline modules/flight_interface/flight_interface.py:6]: Order alphabetically (move down).

File: tests/integration/test_data_merge_worker.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 import numpy as np
 
-from modules import drone_odometry_local
+from modules.common.mavlink.modules import drone_odometry_local
```
Comment (Xierumeng) [inline tests/integration/test_data_merge_worker.py:10]: Order alphabetically (move down).

File: tests/integration/test_geolocation_worker.py [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import numpy as np
 
 from modules import detections_and_time
-from modules import drone_odometry_local
+from modules.common.mavlink.modules import drone_odometry_local
```
Comment (Xierumeng) [inline tests/integration/test_geolocation_worker.py:11]: Order alphabetically (move down).

File: tests/unit/test_geolocation.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 from modules import detection_in_world
 from modules import detections_and_time
-from modules import drone_odometry_local
+from modules.common.mavlink.modules import drone_odometry_local
```
Comment (Xierumeng) [inline tests/unit/test_geolocation.py:10]: Order alphabetically (move down).

**Review by Xierumeng — COMMENTED** [review]
Reviewed.

**Review by maxlou05 — APPROVED** [review]
Approved

**Pull request merged**